### PR TITLE
[Bug Fix] "tooltip is not a function" in analytics V2

### PR DESCRIPTION
**Pull request by alis-khadka**
Addresses: https://github.com/restarone/violet_rails/issues/1609

### Demo Clip

https://github.com/restarone/violet_rails/assets/25191509/1adc7ae1-2596-4ffc-92db-cf6f36bf6356

### CODE REVIEW

**Comment by donrestarone** [comment]
Pending UAT on the review app, there are too many changes to merge directly to staging

**Comment by donrestarone** [comment]
LGTM on mobile, 

image

image

image
Will approve after testing on desktop, and checking the JS console

**Comment by donrestarone** [comment]
The file dropdown only works when in the web settings view because ujs is being loaded twice. Once that issue is fixed, this PR is good to go @Pralish @alis-khadka 

Screen Shot 2023-09-20 at 3 35 49 PM
Screen Shot 2023-09-20 at 3 35 42 PM

File: app/views/layouts/comfy/admin/cms.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 !!!
 %html{lang: I18n.locale}
   = stylesheet_link_tag 'application', media: 'all', 'data-turbo-track': 'reload'
-  = javascript_pack_tag 'trix'
+
```
Comment (donrestarone) [inline app/views/layouts/comfy/admin/cms.html.haml:4]: ⚠️  @Pralish @alis-khadka this causes email composition to break
Screen Shot 2023-09-20 at 3 40 18 PM

**Comment by alis-khadka** [comment]
### New Demo Clip

https://github.com/restarone/violet_rails/assets/25191509/b7e2f622-75b1-46da-8f19-8cab46abd904

**Comment by donrestarone** [comment]
functionality is broken on first visit (eg: file expansion), it only works after refreshing the page. Indicative of Javascript being loaded incorrectly or multiple times

**Comment by alis-khadka** [comment]
@donrestarone 
This is ready for review.

PS: Not sure why the github action for test cases is not running.

**Pull request merged**